pkgs/build-support/trivial-builders/default.nix

```diff
@@ -665,8 +623,8 @@ rec {
       '');
 
 
-  # Write the references (i.e. the runtime dependencies in the Nix store) of `path` to a file.
-
+  # Docs in doc/build-helpers/trivial-build-helpers.chapter.md
+  # See https://nixos.org/manual/nixpkgs/unstable/#trivial-builder-writeReferencesToFile
   writeReferencesToFile = path: runCommand "runtime-deps"
```

Comment:
How is this going to reflect on https://github.com/NixOS/nix/pull/9054?

Reply:
They can be moved back, and something like nixdoc can be added.
For now what matters is that there's no more duplication and contributors know where to add their docs when they change/add things.